submissions/Alissy27/popup/style.css

```diff
@@ -0,0 +1,174 @@
+* {
+    margin: 0;
+    padding: 0;
+    font: 20px Tahoma;
+    color: rgb(11, 83, 38);
+    list-style-type: none;
+    text-decoration: none;
+}
+
+.header {
+    background: rgba(156, 231, 185, 0.27);
+    padding: 52px;
+}
+
+.header__flex{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.header__flex .logo {
+    max-width: 150px;
+    border-radius: 30%;
+    margin-left: 50px;
+}
+
+.nav{
+    display: flex;
+    align-items: center;
+}
+
+.nav__li{
+    margin-left: 15px;
+}
+
+.nav__popup label{
```

Comment:
Avoid styling by HTML tags except if you add base styles. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css) Also, avoid too specific CSS selectors. [Read more about that](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)

Reply:
added class to label, changed css